server/server.go

```diff
@@ -1411,13 +1411,14 @@ func (s *StanServer) buildServerURLs() ([]string, error) {
 		// use 127.0.0.1 or ::1 for host address (important for
 		// Windows since connect with 0.0.0.0 or :: fails).
 		sport := strconv.Itoa(opts.Port)
-		if opts.Host == "0.0.0.0" {
-			hostport = net.JoinHostPort("127.0.0.1", sport)
-		} else if opts.Host == "::" || opts.Host == "[::]" {
-			hostport = net.JoinHostPort("::1", sport)
-		} else {
-			hostport = net.JoinHostPort(opts.Host, sport)
-		}
+    switch opts.Host {
```

Comment:
You need to run "go fmt" because this is not properly formatted (see that our CI caught that).